functions/appInsights.js

```diff
@@ -0,0 +1,15 @@
+module.exports = function (instrumentationKey) {
```

Comment:
Let's cross verify with the Telemetry dashboard requirements and ensure that we are logging all the information necessary to our appinsights instance to generate those dashboards.

Reply:
Had addded contactId & lambdaName. 
After these were added, verified that such dashboards could be built with the info we are pushing.